{
	"title": "CVE-2004-0150",
	"cve": "CVE-2004-0150",
	"notes": [
		{
			"type": "Description",
			"text": "Buffer overflow in the getaddrinfo function in Python 2.2 before 2.2.2, when IPv6 support is disabled, allows remote attackers to execute arbitrary code via an IPv6 address that is obtained using DNS."
		},
		{
			"title": "Published",
			"type": "Other",
			"text": "2004-09-01"
		},
		{
			"title": "Modified",
			"type": "Other",
			"text": "2004-08-17"
		}
	],
	"references": [
		{
			"url": "http://www.securityfocus.com/bid/9836",
			"description": "BID:9836"
		},
		{
			"url": "http://www.debian.org/security/2004/dsa-458",
			"description": "DEBIAN:DSA-458"
		},
		{
			"url": "http://www.gentoo.org/security/en/glsa/glsa-200409-03.xml",
			"description": "GENTOO:GLSA-200409-03"
		},
		{
			"url": "http://www.mandrakesoft.com/security/advisories?name=MDKSA-2004:019",
			"description": "MANDRAKE:MDKSA-2004:019"
		},
		{
			"url": "http://www.osvdb.org/4172",
			"description": "OSVDB:4172"
		},
		{
			"url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/15409",
			"description": "XF:python-getaddrinfo-bo(15409)"
		}
	]
}
